src/Pages/ContentListTable/ContentListTable.tsx

```diff
@@ -240,68 +241,101 @@ const ContentListTable = () => {
   } = data;
 
   const rowActions = useCallback(
-    (rowData: ContentItem): IAction[] =>
-      isRedHatRepository
-        ? features?.snapshots?.accessible
-          ? [
+    (rowData: ContentItem): IAction[] => {
+      const triggerSnapshotAction = {
+        isDisabled: actionTakingPlace || rowData?.status === 'Retrying',
+        title: 'Trigger Snapshot',
+        onClick: async () => {
+          try {
+            await axios.post(`/api/snapshot/${rowData?.uuid}/trigger`);
```

Comment:
as i mentioned [here](https://github.com/content-services/content-sources-frontend/pull/186/files#r1422733023), [here](https://github.com/content-services/content-sources-frontend/pull/186/files#r1425566174), [here](https://github.com/content-services/content-sources-frontend/pull/186#discussion_r1425389227)  and on our video call, api calls need to be done within the  https://github.com/content-services/content-sources-frontend/blob/main/src/services/Content/ContentApi.ts file

Also, this api (`/api/snapshot/${rowData?.uuid}/trigger`) isn't the api call that was implemented in the backend.  The correct one is `POST /api/content_sources/v1.0/repositories/$UUID/snapshot/` which i mentioned [here](https://github.com/content-services/content-sources-frontend/pull/186#discussion_r1425389227)